MinPlatformPkg/Test/Library/TestPointLib/StandaloneMmTestPoint.c

```diff
@@ -0,0 +1,624 @@
+/** @file
```

Comment:
Again, diff'ing this file against SmmTestPoint.c shows very little difference with changes that can be overcome fairly easily like use `gMmSt` for both phases instead of `gSmst` at all. Please consider consolidating the logic.

Reply:
Consolidated.